docs/_static/vdr/v2.yaml

```diff
@@ -231,17 +265,17 @@ paths:
                     ]
         default:
           $ref: '../common/error_response.yaml'
-  /internal/vdr/v2/did/{did}/service/{serviceId}:
+  /internal/vdr/v2/subject/{id}/service/{serviceId}:
     parameters:
-      - name: did
+      - name: id
         in: path
-        description: URL encoded DID.
+        description:  DID Subject.
         required: true
         content:
           plain/text:
             schema:
               type: string
-              example: "did:web:example.com:iam:123"
+              example: "tenant-123"
       - name: serviceId
```

Comment:
🦆 implementers still need to inspect a DID document to find the `serviceId`. So not completely free of DID handling